### ISSUE TITLE
Feature Request: Session Word Count

### ISSUE THREAD

**Issue by AdamIsrael** [AdamIsrael/vscode-author-stats#1]
It'd be nice to have a concept of sessions, so that you can see how many words you've written in a particular session.

This would require:
- commands to manage a session (start, stop, reset)
- new StatusBarItem(s) to display session stats

**Comment by nomad101** [comment]
Yes! This would be a great feature. I'm really missing out on not having a writing session target feature. 

Thanks for creating this extension.
